pages/docs/clients/odbc.html

```diff
@@ -1,7 +1,7 @@
 title: Connecting to Crate with ODBC
 author: Claus Matzinger
 
-The ODBC driver package is not open source but part of our Enterprise License. [Contact us][3] for more details.
+The ODBC driver package is not open source but requires a License for production use. For testing and evaluation a free test license is available. Just sign the test license agreement and provide your details, our team will send you the ODBC driver. [Sign here with Hellosign.](https://www.hellosign.com/s/313c319f)
```

Comment:
"...requires a License for production use..."... What license? By itself this statement is not helpful.